integration_tests/proto_build/Cargo.lock

```diff
@@ -0,0 +1,1040 @@
+# This file is automatically @generated by Cargo.
```

Comment:
if you add this to the rust workspace for integration_tests then it will share a cargo.lock and this 1k line file won't need to exist.

Reply:
error resolved